Frends.FTP.ListFiles/Frends.FTP.ListFiles/Definitions/Result.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System.Collections.Generic;
+
+namespace Frends.FTP.ListFiles.Definitions
+{
+    /// <summary>
+    /// List of files.
+    /// </summary>
+    public class Result
+    {
+        /// <summary>
+        /// List of files.
+        /// </summary>
+        public List<ListObject> Files { get; set; }
```

Comment:
Make setter private (or internal if planning on using object initializer from the comment below about ctor).